roles/ceph-rgw/tasks/multisite/destroy.yml

```diff
@@ -1,38 +0,0 @@
----
```

Comment:
I thought we agreed on moving this to infrastructure-playbooks? Or did I get it wrong?

Reply:
I think we agreed the best course of action is to eventually have an infrastructure playbook and you were going to open up a BZ about it where there can be some discussion.

No matter what, `destroy.yml` isn't being used in master in anyway and having a playbook with similar functionality should come in a separate PR.

Reply:
Right, but let's move it in https://github.com/ceph/ceph-ansible/tree/master/infrastructure-playbooks/untested-by-ci instead of removing it. Thanks

Reply:
Will do.